packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -1,63 +1,58 @@
-import * as React from 'react';
-import { IButtonComponent, IButtonViewProps } from './Button.types';
-import { Text } from '../../Text';
+/** @jsx createElementWrapper */
```

Comment:
what is this for?

Reply:
There is documentation for this in `Slots.tsx`.